mappings/net/minecraft/world/gen/surfacebuilder/SurfaceBuilder.mapping

```diff
@@ -30,16 +30,20 @@ CLASS net/minecraft/class_6724 net/minecraft/world/gen/surfacebuilder/SurfaceBui
 		ARG 3 seaLevel
 		ARG 4 seed
 		ARG 6 randomProvider
-	METHOD method_39102 (Lnet/minecraft/class_6557;IIILnet/minecraft/class_5539;)V
+	METHOD method_39102 buildBadlandsPillar (Lnet/minecraft/class_6557;IIILnet/minecraft/class_5539;)V
+		ARG 1 column
 		ARG 2 x
 		ARG 3 z
 		ARG 4 surfaceY
+		ARG 5 heightLimitView
```

Comment:
I think parameters of `World` superinterfaces are named `world`. Technically a `Chunk` is a `HeightLimitView` but I don't think it's used in that way.

Reply:
Yeah I named it `heightLimitView` instead of `world` as I wasn't certain whether it was just `World`s that were being used here - but if `Chunk`s aren't used I'll go ahead and call it `world`

Reply:
Just checked and it specifically uses a `Chunk`

Reply:
Renamed to `chunk`